### PR TITLE
Add StuFF 2019

### PR DESCRIPTION
**Pull request by GingertronMk1**
Currently:

- [x] Risky Texts
- [x] Order From Chaos
- [x] Decibels
- [x] 2021
- [x] Through No Fault
- [x] Bost-Uni Plues
- [x] Mixtape Mind
- [x] Don't Bother
- [x] Poetry
- [x] Woyzeck
- [x] Ophelia
- [x] Apollo
- [x] Animali
- [x] The Things We Leave Behind
- [x] Decline/Rise
- [x] Hard Truths
- [x] 2 Clowns
- [x] Polly

### CODE REVIEW

**Comment by samozzy** [comment]
Do we think it is worth recording the start times for these shows?

**Comment by BenWoodford76** [comment]
any progress on this?

**Comment by GingertronMk1** [comment]
@BenWoodford76 Not yet, I've been at Dan for the lists but he's obvs been busy with Edinburgh stuff

File: _shows/18_19/2_clowns_1_cup.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+date_end: 2019-06-18
+venue: New Theatre Studio A
+
+cast:
+- name: Grace Gallagher
+  person: false
+- name: Crystal Quinney Barella
+  person: false
+
+crew_incomplete: true
```
Comment (BenWoodford76) [inline _shows/18_19/2_clowns_1_cup.md]: did they have more crew??

Comment (samozzy) [reply _shows/18_19/2_clowns_1_cup.md]: I suspect there was a producer, stage manager and sound designer, but unknown who these people are.

**Comment by samozzy** [comment]
@BenWoodford76 Is anything more missing from this?

**Comment by BenWoodford76** [comment]
@samozzy Will check tomorrow when I have internet again

**Pull request merged**